arm-web/2015-08-01-preview/swagger/logicAppsManagementClient.json

```diff
@@ -0,0 +1,1281 @@
+          {
+            "name": "apiName",
+            "description": "The managed API name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "export",
+            "in": "query",
+            "description": "flag showing whether to export API definition in format specified by Accept header.",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ApiEntity"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/connections": {
+      "get": {
+        "tags": [ "Connections" ],
+        "summary": "Get Connections",
+        "operationId": "Connections_List",
+        "description": "Gets a list of connections.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "Resource Group Name",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "$top",
+            "description": "The number of items to be included in the result.",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "$filter",
+            "description": "The filter to apply on the operation.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ConnectionCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/Connection"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/connections/{connectionName}": {
+      "get": {
+        "tags": [ "Connections" ],
+        "operationId": "Connections_Get",
+        "description": "Gets a connection.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
Nope. All the globally referenced parameters will be properties on the client when AutoRest generates the client.
We are coming up with an extension named `"x-ms-client-property" : true(default)/false` so that global parameter don't end up being properties on the client. Till that happens please specify them explicitly in every operation.